### PR TITLE
🦟 Add whitelist to liquidations

### PR DESCRIPTION
**Pull request by nezouse**

### CODE REVIEW

File: contracts/VesselManagerOperations.sol [hunk 1]
```diff
@@ -110,6 +114,10 @@ contract VesselManagerOperations is IVesselManagerOperations, UUPSUpgradeable, R
 	 * Attempt to liquidate a custom list of vessels provided by the caller.
 	 */
 	function batchLiquidateVessels(address _asset, address[] memory _vesselArray) public override nonReentrant {
+		if(!IAdminContract(adminContract).getRedeemerIsWhitelisted(msg.sender)) {
+			revert VesselManagerOperations__LiquidatorNotWhitelisted();
+		}
+		require(IAdminContract(adminContract).getRedeemerIsWhitelisted(msg.sender), "VesselManagerOperations: Liquidator not whitelisted");
```
Comment (MiksuJak) [inline contracts/VesselManagerOperations.sol]: This is obsolete

**Review by MiksuJak — APPROVED** [review]

File: test/trinity/VesselManagerTest.js [hunk 2]
```diff
@@ -1412,6 +1434,19 @@ contract("VesselManager", async accounts => {
...
 			// --- liquidateVessels() ---
 
+			it("liquidateVessels(): reverts when not whitelisted", async () => {
+				await adminContract.setWhitelistedRedeemer(alice, false)
+	
+				try {
+					const tx = await vesselManagerOperations.liquidateVessels(erc20.address, 2, {from: alice})
+					assert.isFalse(tx.receipt.status)
+				} catch (err) {
+					console.log(err.message)
```
Comment (nezouse) [inline test/trinity/VesselManagerTest.js]:
```suggestion
```

File: test/trinity/VesselManagerTest.js [hunk 3]
```diff
@@ -2442,6 +2477,19 @@ contract("VesselManager", async accounts => {
...
 
 		describe("Batch Liquidations", async () => {
+			it("batchLiquidateVessels(): reverts when not whitelisted", async () => {
+				await adminContract.setWhitelistedRedeemer(alice, false)
+
+				try {
+					const tx = await vesselManagerOperations.batchLiquidateVessels(erc20.address, [], {from: alice})
+					assert.isFalse(tx.receipt.status)
+				} catch (err) {
+					console.log(err.message)
```
Comment (nezouse) [inline test/trinity/VesselManagerTest.js]:
```suggestion
```

**Review by MiksuJak — APPROVED** [review]

**Comment by JustynaBroniszewska** [comment]
We will need 2 separate whitelists. One whitelist that will allow to redeem and put TRI in StabilityPool and the second whitelist for calling liquidate function

**Pull request closed without merging**